Tensile/KernelWriterSource.py

```diff
@@ -3311,7 +3311,13 @@ def wait(self, kernel, tPA, tPB, globalRead, localWrite, localRead, comment):
   # SyncThreads
   ##############################################################################
   def syncThreads(self, kernel, comment=""):
-    return self.indent + self.syncStr + " //" + comment + self.endLine
+    if globalParameters["CurrentISA"] == (9,0,8):
+      return self.indent + self.syncStr + " //" + comment + self.endLine
+    else:
+      if kernel["SubGroup0"]*kernel["SubGroup1"]*kernel["LocalSplitU"] != kernel["WavefrontSize"]:
```

Comment:
I think it is better to follow the same way as KernelWriterAssembly.py
( logically, > should be better than !=)
if kernel["NumThreads"] > self.kernel["WavefrontSize"]

Reply:
We can use kernel["WavefrontSize"] instead of self.kernel["WavefrontSize"]
(self. is redundant...)

Reply:
> 

@nakajee I am not sure if kernel["NumThreads"] includes the value of LocalSplitU.

Reply:
> > 
> 
> @nakajee I am not sure if kernel["NumThreads"] includes the value of LocalSplitU.

@nakajee I checked the code again, kernel["NumThreads"] includes parameter LocalSplitU.